clog/loggers.py

```diff
@@ -36,9 +36,13 @@
 import pkg_resources
 
 import thriftpy
+
+# Will be set to True if the Monk dependency is installed, False otherwise
```

Comment:
Not sure this comment is really needed, but that should be fine, heh 😄

Reply:
Yeah, not really needed. I'll remove it.